### PR TITLE
add task and resume about code competence 1, clean code and introduction react.

### PR DESCRIPTION
**Pull request by BagusAri-17**
Hello sir, I have completed the assignment for 
1. Code Competence 1.
2. Clean Code.
3. Introduction React. 

Please review the code I have created. Thank you.

### CODE REVIEW

File: 10_clean-code/praktikum/createProduct.html [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+                // Menambahkan data pada table
+                const dataTableBody = document.getElementById('table-product');
+                const newRow = dataTableBody.insertRow();
+
+                const cell1 = newRow.insertCell(0);
+                const cell2 = newRow.insertCell(1);
+                const cell3 = newRow.insertCell(2);
+                const cell4 = newRow.insertCell(3);
+                const cell5 = newRow.insertCell(4);
+                const cell6 = newRow.insertCell(5);
```
Comment (syukronarie) [inline 10_clean-code/praktikum/createProduct.html:245]: naming variable harus descriptive

File: 10_clean-code/praktikum/createProduct.html [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+
+                cell1.textContent = productName;
+                cell2.textContent = productCategory;
+                cell3.innerHTML = '<img src="' + imageUrl + '" width="50" height="50" />';
```
Comment (syukronarie) [inline 10_clean-code/praktikum/createProduct.html:249]: pakai innerChild seharusnya

**Review by syukronarie — APPROVED** [review]

**Review by syukronarie — APPROVED** [review]

File: 10_clean-code/praktikum/createProduct.html (hunk 1 above)
Comment (BagusAri-17) [reply 10_clean-code/praktikum/createProduct.html:245]: Baik mas, akan saya perbaiki.

File: 10_clean-code/praktikum/createProduct.html (hunk 2 above)
Comment (BagusAri-17) [reply 10_clean-code/praktikum/createProduct.html:249]: Baik mas, akan saya perbaiki.

**Pull request merged**